### ISSUE TITLE
ObjectGroup.baseTypeOfCollection - change from string to array for baseTypeOfCollection

### ISSUE THREAD

**Issue by jbstatgen** [tdwg/cd#446]
https://github.com/tdwg/cd/blob/79070c8bbff1959d4f534290decd8bc3d638a03c/standard/json-schema/object-group.json#L18

Object groups might include entities of different basic types, e.g. physical objects (ie. MaterialEntities) and their associated photographs (ie. InformationArtifacts). This is especially the case for derived, compound object groups representing, eg. organization-wide collections.

Thus, we might consider to change the expected data type to be `array` instead of `string`. See eg. `collectionManagementSystem`, which is also a property within `ltc:ObjectGroup` and has its data type set to `array`.

**Comment by essvee** [comment]
Discussed at [16th Feb meeting](https://docs.google.com/document/d/16RCOisUhdl3h9ZmV2tZFHaRxnzdSfKbQKXKEaFKFAdA/edit#bookmark=id.joyonp3mzvz). 

tl;dr: every other field in ObjectGroup is an array except for this one, `description` and `collectionName` so fundamentally it makes no sense to have `baseTypeOfCollection` standing alone as a single-value field, so we should change this to being an array fo strings. BUT, we need to make it clear in docs/recipes that if you model your OG with a lot of arrays, you're not going to be able to quantify any of those characteristics.

**Comment by fmjjones** [comment]
Can/Should we include this recipe in the CollectionDescriptionScheme Class or at least cross reference?

**Comment by wouteraddink** [comment]
Looking at the given examples (material samples, information artefacts), an array would make sense.

**Comment by jbstatgen** [comment]
Updated to repeatable = Yes in csv.
